### DISCUSSION TITLE
Is it possible to have an offset for an ordinal field?

### DISCUSSION THREAD

**Discussion by elmehalawi** [vega/vega-lite#7509] [Q&A] [answered]
I have [a graph](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAKyUAVhDYA7EABoQAEySYUqUMSSCGcCGgDaodRADWaEAEFFICOoBOmNAEYlcWcrQBmJZgCeOHCWOA5sknBQmHBeAL4KJkjmlgBCtvZITj7unmgAHH6Bweih4ZHRcQkgphboAMLpjs6oBSAeXqguAAyFQSFhEVExIPGJyeg2ShlZqABMOR0ALL3FIEhRDDojVTWpjZnNy225qACcK5brmJuC22O1IA1TTWgA7AuubtVFlxtbo9Ukg9JiBtLpil81Nlvn10AwcHdAeNrLYwXo0D01ocLnCEQDdhNUTp0Z0lFDUAA2HEgeGIgkopRo4qY8nnGGrWn4oGWEFM1xk5rdamcnbc9BpRnEiEC6EBWE0vGi5ES0FSjEy1BHOUcxX3PaS8H8rFoKnsywivXiomG1As5ps7Xm3VIh4qvmk42dTGO3EjAC68RAAHcaMp6Pkupj4DQyFgfJGclA2MoaLIyGhQAAPDNm9AAR02skwdDUNFII0DgiQ-jgDiMoGQDlqJh+6CYmVs7YgcGEslWCFDykEwUDHiTKbTOf8OYAZjQex0XbYfSA2A4J-8lNnNCA5wvLNNnNSC0giyXi+XA5nZrP54JF+0K34HKeIDO1wh67uaIJonWd3u96WI67gnrcJQDOUwyxLEAZVI2za5iA7Z1ko0axocmJJoIa6WA4wxKF2Papv2g7Dtsxzjqm6Y7tOAF3ouuzUmuG63Fet77ugh7Lq2IAnme6gXiOW43vRnHHHEgaYC+shvh+X5zr+tYcUB6AgW0YG-Nc-ywQCCE5iuvbBEo2G4SUDAODg5GjrIVGTju26gIBi5QoGdFOQxwHcsx66pjpz6vu+Difqgxjfkp-4eeJ6lwJpvowVUGRQA8TEgGQYTwkwdGGEufpkgY6D+AA8jOM7ds4AJQDoUAMFW0SWKo1wIJQxWleVthJJY066QGQA) that is similar to a gantt chart. I'm trying overlay a line graph on top of each "row" in the gantt. Basically a line graph whose center is each ordinal point.

I tried several things, including trying to "cast" the ordinal y value to a variable and using the `calculate` transform to add an offset based on the value. But vega doesn't accept that since you can't use `"type": "ordinal"` in a transform clause.

Is this possible with vega?

**Comment by domoritz** [comment, marked as answer]
Are you looking for https://github.com/vega/vega-lite/issues/4703?
